srcpkgs/HeroicGamesLauncher/template

```diff
@@ -0,0 +1,53 @@
+# Template file for 'HeroicGamesLauncher'
+pkgname=HeroicGamesLauncher
+version=2.8.0
+revision=1
+archs="x86_64*"
+_electron_version=24
+hostmakedepends="electron${_electron_version} jq nodejs python3 yarn"
+depends="electron${_electron_version} legendary heroic-gogdl"
+checkdepends="tar xz"
+short_desc="Open source launcher for GOG and Epic Games"
+maintainer="sirkhancision <jsantiago12tone@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher"
+distfiles="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/archive/v${version}.tar.gz"
+checksum=59f068d6ba7e6c9c5d3c08f06b4cee23f07452777b9edb8baa61895fa9f40320
+nopie_files="/usr/lib/HeroicGamesLauncher/resources/app.asar.unpacked/build/bin/linux/gogdl
+/usr/lib/HeroicGamesLauncher/resources/app.asar.unpacked/build/bin/linux/legendary"
```

Comment:
Indent here with one space when splitting values in double quotes into multiple lines.